include/seqan3/alphabet/quality/phred94.hpp

```diff
@@ -0,0 +1,119 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Lydia Buntrock <lydia.buntrock AT fu-berlin.de>
+ * \brief Provides seqan3::phred94 quality scores.
+ */
+
+#pragma once
+
+#include <seqan3/alphabet/quality/quality_base.hpp>
+
+// ------------------------------------------------------------------
+// phred94
+// ------------------------------------------------------------------
+
```

Comment:
```suggestion
```

Since there is nothing else in this file we don't need to add such separations I think :)